### DISCUSSION TITLE
YouTube videos get the wrong filename

### DISCUSSION THREAD

**Discussion by marcoagpinto** [aclap-dev/video-downloadhelper#1386] [Help and Q&A]
### Your question

Heya,

I am having issues with YouTube videos.

They get the wrong filename, sometimes a URL other times the filename of the previous download.

### Addon & CoApp version and details

I have the latest Firefox and the latest add-on on Mozilla's site.

**Comment by rda1267** [comment]
I am having the same problem.   only way i can get around is to copy the url, close foxfire and come back in.
paste url and it will get 1 right, then have to repeat.   kinda a pain.  hope someone can fix
thanks
Author
[mig](https://addons.mozilla.org/en-US/firefox/user/32479/?utm_source=firefox-browser&utm_medium=firefox-browser&utm_content=addons-manager-user-profile-link)
Version
9.1.0.44
Last Updated
September 10, 2024

**Reply by Richard8263** [reply]
This is my very first post on github. Although I'd considered creating an account for some time, I did so yesterday specifically to provide feedback on this topic to the software creators.
I'm having the same, or a very similar issue. Only on YouTube, as far as I've noticed, the first download of the day (or perhaps since my last computer reboot?) will be properly titled, matching the video title posted on YouTube. But for subsequent downloads, totally unrelated videos will be titled exactly the same as the first one. I can't swear that this happens every time, but it has happened _multiple_ times; probably more often than not.
